pkg/registry/generic/rest/streamer.go

```diff
@@ -56,6 +59,16 @@ func (s *LocationStreamer) InputStream(apiVersion, acceptHeader string) (stream
 	if err != nil {
 		return
 	}
+
+	if resp.StatusCode == http.StatusInternalServerError {
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
OK I can get on board with that.

@feihujiang is this enough info for you to continue with this?